src/main/java/org/openstreetmap/atlas/geography/atlas/command/PBFToAtlasSubCommand.java

```diff
@@ -0,0 +1,213 @@
+    public void usage(final PrintStream writer)
+    {
+        writer.println("-pbf=/path/to/pbf : pbf to convert");
+        writer.println("-output=/path/to/output/atlas : Atlas file to output to");
+        writer.println("-edge-filter=/path/to/json/edge/filter : json filter to determine Edges");
+        writer.println("-node-filter=/path/to/json/node/filter : json filter for OSM nodes");
+        writer.println(
+                "-relation-filter=/path/to/json/relation/filter : json filter for OSM relations");
+        writer.println("-way-filter=/path/to/json/way/filter : json filter for OSM ways");
+        writer.println("-load-relations=boolean : whether to load Relations; defaults to true");
+        writer.println("-load-ways=boolean : whether to load ways; defaults to true");
+        writer.println(
+                "-country-codes=list,of,ISO3,codes : countries from the country map to convert");
+        writer.println(
+                "-country-boundary-map=/path/to/WKT/or/shp : a WKT or shp file containing a country boundary map");
+        writer.println(
+                "-country-slicing=boolean : whether to perform country slicing; defaults to true");
+        writer.println(
+                "-way-sectioning=boolean : whether to perform way sectioning; defaults to true");
+        writer.println(
+                "-way-section-filter=/path/to/json/way/section/filter : json filter to determine where to way section");
+    }
+
+    @Override
+    public int execute(final CommandMap map)
+    {
+        new OsmPbfLoader((File) map.get(INPUT_PARAMETER), getAtlasLoadingOption(map))
+                .saveAtlas((File) map.get(OUTPUT_PARAMETER));
+        return 0;
+    }
+
+    /**
+     * Get or create a {@link CountryBoundaryMap}. If the country-boundary-map parameter is set,
+     * this will attempt to load the text or shape file from that parameter. Else, this will load
+     * using the entire world as the country UNK (unknown).
+     *
+     * @param map
+     *            {@link CommandMap} containing the {@code COUNTRY_MAP_PARAMETER}
+     * @return {@link CountryBoundaryMap} loaded from a file or default
+     */
+    private CountryBoundaryMap getCountryBoundaryMap(final CommandMap map)
+    {
+        final Optional<File> countryMapOption = (Optional<File>) map
+                .getOption(COUNTRY_MAP_PARAMETER);
+        CountryBoundaryMap countryMap = CountryBoundaryMap
+                .fromBoundaryMap(Collections.singletonMap("UNK", MultiPolygon.MAXIMUM));
+        if (countryMapOption.isPresent())
+        {
+            if (FilenameUtils.isExtension(countryMapOption.get().getName(), "txt"))
+            {
+                countryMap = CountryBoundaryMap.fromPlainText(countryMapOption.get());
+            }
+            else if (FilenameUtils.isExtension(countryMapOption.get().getName(), "shp"))
+            {
+                countryMap = CountryBoundaryMap.fromShapeFile(countryMapOption.get().getFile());
```

Comment:
It looks like `countryMapOption.get()` is fetched multiple times. Could make use of a variable for this file.